pom.xml

```diff
@@ -405,6 +406,15 @@
     </scm>
 
     <profiles>
+        <profile>
+            <id>java11-release8</id>
+            <activation>
+                <jdk>[11,</jdk>
```

Comment:
This is not a valid range https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

Should be `[11,)`